src/client/handlers/agentStatus.ts

```diff
@@ -20,12 +21,8 @@ class AgentStatusHandler extends UnaryHandler<
       nodeIdEncoded: nodesUtils.encodeNodeId(pkAgent.keyRing.getNodeId()),
       clientHost: pkAgent.webSocketServerClient.getHost(),
       clientPort: pkAgent.webSocketServerClient.getPort(),
-      proxyHost: pkAgent.proxy.getProxyHost(),
-      proxyPort: pkAgent.proxy.getProxyPort(),
-      agentHost: pkAgent.grpcServerAgent.getHost(),
-      agentPort: pkAgent.grpcServerAgent.getPort(),
-      forwardHost: pkAgent.proxy.getForwardHost(),
-      forwardPort: pkAgent.proxy.getForwardPort(),
+      agentHost: pkAgent.quicSocket.host as unknown as Host,
+      agentPort: pkAgent.quicSocket.port as unknown as Port,
```

Comment:
Yes, the `js-quic` `Host` type and the `Polykey` `Host` type don't overlap. I'll have to got through this and fix up some types so this is not needed.

I don't think there was any decision around it? I think this is just how you implemented getting the host and port in `js-quic`.

Reply:
Yes, so based on discussion with Amy, we should change `js-quic` on the boundary input of the program to take as `string` and `number` instead. And just do the verification there of them being a proper host or port number. We would need additional exceptions to be thrown in js-quic for that. It would be similar to parse exceptions. See: https://github.com/MatrixAI/Polykey-Docs/issues/20.

I think js-quic can be updated to support that.

Reply:
We discussed this today regarding `toHost/fromHost` and `toPort/fromPort` in js-quic. Did you update the library and integrate the changes here? Taking care to manage exceptions too?

If so, please resolve and explain what you changed.

Reply:
For providing the host or port we now use `sting ` and `number`. This is the same to getting these values. Internally we're still using `Host` and `Port`.